randovania/games/super_metroid/logic_database/Brinstar.txt

```diff
@@ -672,15 +675,22 @@ Extra - asset_id: 92
   * Layers: default
   * Normal Door to Big Pink/Door to Green Hill Zone
   > Door to Morph Ball Room
-      Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+      Any of the following:
+          Hi-Jump Boots or Space Jump or Wall Jumps (Beginner)
+          # IBJ
```

Comment:
you can also do this with a diagonal one.

Reply:
I really don't think its necessary, but I'll add it. I think that in every case if IBJ is an option, then it replaces HBJ. They require the same items and one is extremely difficult to do.

Reply:
you cant do HBJ everywhere where you can do IBJ, so this can be used to limit logic in a certain way.